### ISSUE TITLE
Index > 0 as prop doesn't actually render on a different index

### ISSUE THREAD

**Issue by fungilation** [leecade/react-native-swiper#114]
Example Swiper component in render():
```
<Swiper
        style={styles.wrapper}
        ref={(c) => { this.AppSwiper = c }}
        index=initIndex
        loop={true}
        bounces={true}
        horizontal={true}
        showsButtons={false}
        keyboardDismissMode='on-drag'
        directionalLockEnabled={true}
        activeDot={
          <View style={{backgroundColor: 'rgba(255, 116, 115, 0.95)', width: 8, height: 8, borderRadius: 4, marginLeft: 3, marginRight: 3, marginTop: 3, marginBottom: 3}} />
}>
...
</Swiper>
```

With 3 ```<View>```s as children to Swiper, when initIndex is hardcoded to 1, the Swiper does not render with 2nd View shown as should be intended. It always render the 1st View, regardless of number set for the index prop.

**Comment by gnmonsalve** [comment]
+1

**Comment by LiuXingQiPan** [comment]
+1

**Comment by Slebee** [comment]
+1

**Comment by jpmerritt** [comment]
I'm hitting this issue as well.

**Comment by jpmerritt** [comment]
This is fixed in the 1.4.4 release.

**Comment by fungilation** [comment]
Fix confirmed.
